run_dir/design/flowcells.html

```diff
@@ -50,7 +50,7 @@ <h1><span id="page_title">Flowcells</span> </h1>
                 <td class="run_id">
                     <a href="/flowcells/{{ onefc }}">{{ flowcells[onefc]["run id"] }}</a>
                     {% if flowcells.get(onefc).get('demultiplexing') == 'Done' %}
-                        <abbr title="Demultiplexing completed">&nbsp;<span class="glyphicon glyphicon-ok-sign"></span></abbr>
+                        <span class="glyphicon glyphicon-ok-sign"></span>
```

Comment:
I quite like these - the dashed lines aren't of importance, but hovering should give a question mark cursor and then some text explaining what the icon means. We could also make it a [proper tooltip](https://getbootstrap.com/docs/3.3/javascript/#tooltips) which would be even better.